### ISSUE TITLE
Remove metricsContext payload from endpoints where it is never sent or used

### ISSUE THREAD

**Issue by philbooth** [mozilla/fxa-auth-server#1345]
Sometimes `service` is passed as a query parameter and sometimes it is passed in the payload. Recently, [when I added the `metricsContext` stuff](https://github.com/mozilla/fxa-auth-server/pull/1213), I invented yet another way to pass the `service` via the `metricsContext.service` payload property.

For some reason, this never struck me as confusing until I looked at the code for `fxa-js-client` just now. Maybe it's too late to do something about it or maybe the risk of confusion isn't great enough to justify spending time on this but I wanted to ask the question.

What, if anything, should we do about these different `service` params?

**Comment by shane-tomlinson** [comment]
> What, if anything, should we do about these different service params?

Come up with one true way. :D

**Comment by philbooth** [comment]
In a f2f conversation with @shane-tomlinson just now, we agreed that I was misguided when I added `metricsContext` to the payloads of some auth server endpoints. It is only actually used from `/account/create` and `/account/login` (and the content server only passes it to those endpoints as well). I added it to the others because I kind of thought that any endpoint which emits an activity event should be able to have its metrics context overridden by the caller. That seems pointless now.

So, I'm renaming this issue from `Redundancy of request.query.service, request.payload.service, request.payload.metricsContext.service` to `Remove metricsContext payload from endpoints where it is never sent or used`.
